Codeinwp/feedzy-rss-feeds
# feat: add banner support

### Summary


Add SDK banner support

### Will affect visual aspect of the product

YES

### Screenshots 

![image](https://github.com/user-attachments/assets/71564d09-326d-4060-90d3-d334b4925edb)

![image](https://github.com/user-attachments/assets/e016b612-ce96-4212-a939-c01068902beb)

![image](https://github.com/user-attachments/assets/94062e9e-ed06-44ab-82e6-353acb3dd0b1)



### Test instructions


Must be used with https://github.com/Codeinwp/themeisle-sdk/pull/269

- Set the instance date between 25 NOV and 3 DEC to trigger Black Friday banner
- You should see the banner on the Library page.
- You should NOT see the banner if it is not a Blackfriday Interval

## Check before Pull Request is ready:



Closes https://github.com/Codeinwp/themeisle/issues/1688
